test/shutdown-testing.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+set -eu
+
+declare appPID
+
+_get_app_pid() {
+  appPID=($(jps | grep ApplicationKt))
```

Comment:
This will return just the pid.

appPID="$(jps -l | grep edu.wgu.osmt.ApplicationKt | awk '{print $1}')"